README.md

```diff
@@ -524,3 +525,12 @@ Allows to leash villagers
 - Default value: `true`  
 - Suggested options: `true`, `false`
 - Categories: `SURVIVAL`, `FEATURE`, `REDCRAFT`
+
+## playerme
+
+Allows deopped player to run /player on themselves
+
+- Type: `boolean`  
+- Default value: `true`  
+- Suggested options: `true`, `false`
+- Categories: `SURVIVAL`, `FEATURE`, `REDCRAFT`, `COMMAND`  
```

Comment:
Da posizionare nel punto corretto, al fine che le regole siano disposte in ordine alfabetico

Reply:
da rinominare in commandPlayerme per consistenza